hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -3971,15 +3971,15 @@
       used by map stores and jdbc sinks and sources
       -->
     <data-link name="mysql-database">
-        <class-name>com.hazelcast.datalink.JdbcDataLink</class-name>
+        <type>com.hazelcast.datalink.impl.JdbcDataLink</type>
```

Comment:
minor: should we use a shorter type name?

Reply:
Yes

2463c5d1464